mock/py/mock.py

```diff
@@ -397,6 +399,20 @@ def command_parse():
     return (options, args)
 
 
+def handle_signals(buildroot, number, frame):
+    signal = ""
+    if number == 1:
+        signal = "SIGHUP"
+    elif number == 13:
+        signal = "SIGPIPE"
+    elif number == 15:
+        signal = "SIGTERM"
```

Comment:
This can be better done as
```
signal_names = {1: "SIGHUP", 
               13, "SIGPIPE",
...
}
#all this can be defined at the top of file. and then
log.info("\nReceived signal {} activating orphansKill".format(signal_name[number]))
```
btw original name of variable `signal` override the name of the module. Not a good idea.